src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -410,6 +410,7 @@ def __init__(
         env_vars=None,
         workload_profile_name=None,
         ingress=None,
+        use_default_container_registry=None
```

Comment:
Please consider renaming to `force_single_container_updates` or something of that kind.
Here `use_default_container_registry` won't work once we have BYO ACR. Also, it would make more sense in the containerapp_decorator.py to have something like `force_single_container_updates` (the default container registry is not a concern of the decorator)

Reply:
Updated.